src/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -90,7 +90,7 @@ module ocn_forward_mode
    use ocn_equation_of_state
 
    use ocn_vmix
-   use ocn_vmix_gotm, only: ocn_vmix_gotm_finalize
+   !use ocn_vmix_gotm, only: ocn_vmix_gotm_finalize
```

Comment:
Do you plan to use it again once the GOTM license issue is addressed? If not, maybe we just delete this line for cleanliness?